tendrl/gluster_integration/sds_sync/brick_utilization.py

```diff
@@ -0,0 +1,155 @@
+    mountPoints = {}
+    with open('/proc/mounts', 'r') as f:
+        for line in f:
+            if line.startswith("/") or not filter:
+                mount = {}
+                tokens = line.split()
+                mount['device'] = tokens[0]
+                mount['fsType'] = tokens[2]
+                mount['mountOptions'] = tokens[3]
+                mountPoints[tokens[1]] = mount
+    return mountPoints
+
+
+def _getStats(mountPoint):
+    data = os.statvfs(mountPoint)
+    total = (data.f_blocks * data.f_bsize) / ONE_GB_BYTES
+    free = (data.f_bfree * data.f_bsize) / ONE_GB_BYTES
+    used_percent = 100 - (100.0 * free / total)
+    total_inode = data.f_files
+    free_inode = data.f_ffree
+    used_percent_inode = 100 - (100.0 * free_inode / total_inode)
+    used = total - free
+    used_inode = total_inode - free_inode
+    return {'total': total,
+            'free': free,
+            'used_percent': used_percent,
+            'total_inode': total_inode,
+            'free_inode': free_inode,
+            'used_inode': used_inode,
+            'used': used,
+            'used_percent_inode': used_percent_inode}
+
+
+def getLvs():
+    lvmCommand = ("lvm vgs --unquoted --noheading --nameprefixes "
+                 "--separator $ --nosuffix --units m -o lv_uuid,"
+                 "lv_name,data_percent,pool_lv,lv_attr,lv_size,"
+                 "lv_path,lv_metadata_size,metadata_percent,vg_name")
+    cmd = cmd_utils.Command(lvmCommand, True)
+    out, err, rc = cmd.run()
+    if rc != 0:
+        Event(
+            Message(
+                priority="error",
+                publisher=NS.publisher_id,
+                payload={"message": str(err)}
+            )
+        )
+        return None
+    out = out.split('\n')
+    l = map(lambda x: dict(x),
+            map(lambda x: [e.split('=') for e in x],
+                map(lambda x: x.strip().split('$'), out)))
+
+    d = {}
+    for i in l:
+        if i['LVM2_LV_ATTR'][0] == 't':
+            k = "%s/%s" % (i['LVM2_VG_NAME'], i['LVM2_LV_NAME'])
+        else:
+            k = os.path.realpath(i['LVM2_LV_PATH'])
+        d.update({k: i})
+    return d
+
+
+def getMountStats(mount_path):
+    def _getMounts(mount_path=[]):
+        mount_list = map(_getMountPoint, mount_path)
+        mountPoints = _parseProcMounts()
+        outList = set(mountPoints).intersection(set(mount_list))
+        # list comprehension to build dictionary does not work in python 2.6.6
+        mounts = {}
+        for k in outList:
+            mounts[k] = mountPoints[k]
+        return mounts
+
+    def _getThinPoolStat(device):
+        out = {'thinpool_size': None,
+               'thinpool_used_percent': None,
+               'metadata_size': None,
+               'metadata_used_percent': None,
+               'thinpool_free': None,
+               'metadata_free': None,
+               'thinpool_used': None,
+               'metadata_used': None}
+
+        if lvs and device in lvs and \
```

Comment:
Here lvm is identified https://github.com/Tendrl/gluster-integration/pull/244/files#diff-d85a92b3b059b22fbf7b2530c58f19f3R122